docs/docs/advanced/container/unpacked-filesystem.md

```diff
@@ -114,3 +114,10 @@ Total: 20 (UNKNOWN: 0, LOW: 2, MEDIUM: 10, HIGH: 8, CRITICAL: 0)
 ```
 
 </details>
+
+!!! warning
+    This method may show false positives for RHEL-based images that use Red Hat's own security advisories (RHEL, CentOS, etc.).
+    
+    This is due to `content_sets`, which can be different for multiple layers.
+    But when all files from layers are combined, we cannot correctly match packages with their `content_sets`.
+    
```

Comment:
This is not specific to `docker export`, but more like a limitation of `trivy rootfs` (and `trivy vm`).
In all environments where `trivy rootfs` runs, such as [embedded in dockerfile](https://trivy.dev/latest/docs/advanced/container/embed-in-dockerfile/), layer information is typically missing. This means that content manifests may not be processed correctly. Therefore, I'd suggest documenting it in [rootfs](https://github.com/aquasecurity/trivy/blob/d99c18aa6bc5b6668cb4f8565f780851a4a942ba/docs/docs/target/rootfs.md) and [VM](https://github.com/aquasecurity/trivy/blob/d99c18aa6bc5b6668cb4f8565f780851a4a942ba/docs/docs/target/vm.md).

Reply:
I also think about [RedHat](https://trivy.dev/latest/docs/coverage/os/rhel/) and [centOS](https://trivy.dev/latest/docs/coverage/os/centos/) pages. 

All the same, this only applies to these OSes.
Maybe it makes sense to write it there.
What do you think?

Reply:
It sounds good to me. Also, we may want to add a link in VM and rootfs, like `Scanning Red Hat has a limitation, see the Red Hat page for details.

By the way, CentOS doesn't have content manifests anyway. Scanning images and unpacked rootfs lead to the same result.

Reply:
I moved (and added more info) info into rhel page - https://github.com/aquasecurity/trivy/pull/8820/commits/e92ba724599eba3039346749241fcac8c108218c
And added links for rootfs and vm - https://github.com/aquasecurity/trivy/pull/8820/commits/7703f14371a40be04440d9d0d23b02d12177a20f